cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -256,12 +277,21 @@ def test_vmware_inaccessible_datastore_vm_provisioning():
             2.See all available templates
             3.CFME should provision VM on datastore other than the one that is inaccessible.
     """
-    pass
+    inaccessible_datastores = [
+        datastore for datastore in provider.mgmt.list_datastore()
+        if not provider.mgmt.get_datastore(datastore).summary.accessible]
+    if inaccessible_datastores:
+        logger.info("Found {} inaccessible_datastores".format(inaccessible_datastores))
+    else:
+        pytest.skip("This provider {} has no inaccessible_datastores.".format(provider.name))
+    vm = appliance.collections.infra_vms.create('test-vmware-{}'.format(
+        fauxfactory.gen_alphanumeric()), provider, find_in_cfme=True, wait=True,
+        form_values={'environment': {'automatic_placement': True}})
+    assert vm.datastore.name not in inaccessible_datastores
```

Comment:
**Required** There should be cleanup of this VM after this test

Reply:
Yeah totally forgot it seems. Let me fix that.